Source/WebCore/Modules/cookie-store/CookieStore.cpp

```diff
@@ -272,7 +272,7 @@ void CookieStore::get(CookieStoreGetOptions&& options, Ref<DeferredPromise>&& pr
         promise->resolve<IDLDictionary<CookieListItem>>(CookieListItem(WTFMove(cookies[0])));
     };
 
-    m_mainThreadBridge->get(WTFMove(options), WTFMove(completionHandler));
+    Ref { m_mainThreadBridge }->get(WTFMove(options), WTFMove(completionHandler));
```

Comment:
Rather than keep repeating `Ref { m_mainThreadBridge }`, we should add protectedMainThreadBridge function which returns a Ref.